pkgs/development/python-modules/pyezminc/default.nix

```diff
@@ -0,0 +1,36 @@
+{ buildPythonPackage, isPy3k, fetchFromGitHub, stdenv,
+  netcdf, hdf5, libminc, ezminc,
+  cython, numpy, scipy
+}:
+
+buildPythonPackage rec {
+  pname = "pyezminc";
+  version = "1.2";
+  name = "${pname}-${version}";
+ 
+  disabled = isPy3k;  # ??
```

Comment:
README says

> BUILD REQUIREMENTS:
> - python2-devel

so I think it should remain disabled for Python 3. Please update the comment.

Reply:
comment removed